packages/plugin/src/plugin/plugin-context.ts

```diff
@@ -14,10 +14,12 @@ import { CancellationTokenSource } from '@theia/core/lib/common/cancellation';
 import { QuickOpenExtImpl } from './quick-open';
 import { Plugin } from '../api/plugin-api';
 import { getPluginId } from '../common/plugin-protocol';
+import { MessageRegistryExt } from './message-registry';
 
 export function createAPI(rpc: RPCProtocol): typeof theia {
     const commandRegistryExt = rpc.set(MAIN_RPC_CONTEXT.COMMAND_REGISTRY_EXT, new CommandRegistryImpl(rpc));
     const quickOpenExt = rpc.set(MAIN_RPC_CONTEXT.QUICK_OPEN_EXT, new QuickOpenExtImpl(rpc));
+    const messageRegistryExt = rpc.set(MAIN_RPC_CONTEXT.MESSAGE_REGISTRY_EXT, new MessageRegistryExt(rpc));
```

Comment:
You don't need to do `rpc.set`, as your `MessageRegistryExt` is never called from Theia core part

Reply:
Ok. I just added a mechanism for possible future In the first realization. I'll remove.

Reply:
done